lib/ui5Framework/Sapui5MavenSnapshotResolver.js

```diff
@@ -208,6 +208,13 @@ class Sapui5MavenSnapshotResolver extends AbstractResolver {
 			}
 		}
 
+		if (!process.stdout.isTTY) {
+			// Do not prompt if stdout is non-interactive (i.e. in CI environments) and just use the URL
```

Comment:
Isn't this against the idea to first ask for confirmation to use the URL?

Reply:
Yeah, I was asking myself that too. Since we can't ask the user in this case, we need to decide upfront.

I went with using the Maven config since I expect this to make things easier in CI environments where the config is already present (I know of some base images where this is the case). But you're right, it might be unwanted or unexpected.

However, the only alternative I see is to always ignore the Maven configuration in CI environments, forcing users to configure the environment variable or ui5 config.